### PR TITLE
Adding a simple jupyter notebook to examples.

### PR DESCRIPTION
**Pull request by bazkiaei**
This notebook includes the main steps of creating a mock image. There are some initial steps needed to prepare the simulation data before creating the mock image, all these steps have been removed from the notebook and replaced by one function, `mocks.prepare_data`. To make it easy, some functions are added to the utils and this function uses them to prepare the data. These steps might be redundant when we have data from Genesis simulations but for now, we need them.

### CODE REVIEW

File: mocks/mocks.py [hunk 1]
```diff
@@ -21,6 +21,23 @@
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
+from mocks import utils
+
+
+def prepare_data(particle_pos,
```
Comment (bazkiaei) [inline mocks/mocks.py]: This function prepares the data and it is possible that the data which Chris will provide for us does not need to be prepared.

File: mocks/mocks.py [hunk 2]
```diff
@@ -21,6 +21,23 @@
...
+                 particle_mass,
+                 z_range=[35.69, 35.80] * u.Mpc,
```
Comment (bazkiaei) [inline mocks/mocks.py]: I know I am doing hardcoding here, but I leave it like this for now.

**Comment by lspitler** [comment]
@bazkiaei looking good. Here are some comments about the jupyter notebook:

- [ ] `parse_config` takes `sim_pc_pixel` as input. Maybe rename to `sim_pc_particle` as the simulation does not have pixels, but particles.
- [ ] instead of `cosmo = mocks.create_cosmology(config)` would it make sense to just create a new entry in the `config` dictionary within `mocks.parse_config` that has the cosmology: `config["cosmology"] = mocks.create_cosmology(config`.
- [ ] same comment as the previous one for this line: `redshift = mocks.compute_redshift(config['galaxy_distance'], cosmo)`
- [ ] in fact, looks like `config['pixel_scale'] = mocks.compute_pixel_scale(redshift,`, `config['total_mag'] = apparent_ABmag`, `imagers = imager.create_imagers(), huntsman = imagers['canon_sbig_dark']` all can be put into `parse_config` to simplify the startup signficantly.
- [x] `'canon_sbig_dark'` should be a parameter in the config file.
- [ ] `convolution_boundary='extend'` is it needed or will it default to extend?
- [x] good to show one noisy image before stacking
- [x] suggest defining `yrange=1200:2100 xrange=800:1700` somewhere up top so the user can quickly change the range for all plots
- [x] maybe example "Creating a simple projected image from the data." isn't necessary?

File: mocks/mocks.py (hunk 1 above)
Comment (lspitler) [reply mocks/mocks.py]: maybe rename to 
```suggestion
def crop_simulation_data(particle_pos,
```

**Review by lspitler — APPROVED** [review]
@bazkiaei good. See my comment about the jupyter notebook.

**Pull request merged**